packages/firebase_remote_config/lib/src/remote_config_value.dart

```diff
@@ -0,0 +1,58 @@
+part of firebase_remote_config;
+
+/// ValueSource defines the possible sources of a config parameter value.
+enum ValueSource { valueStatic, valueDefault, valueRemote }
+
+/// RemoteConfigValue encapsulates the value and source of a Remote Config
+/// parameter.
+class RemoteConfigValue {
+  List<int> _value;
+  ValueSource _source;
+
+  RemoteConfigValue._(this._value, this._source);
+
+  ValueSource get source => _source == ValueSource.valueDefault
```

Comment:
Add a dartdoc comment here, e.g. "Indicates at which source this value came from."

Reply:
Done